### PR TITLE
SDI-505 Track ids of all Nomis entities created

### PR DESCRIPTION
**Pull request by mikehalmamoj**

### CODE REVIEW

File: src/test/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/activities/ActivityServiceTest.kt [hunk 1]
```diff
@@ -147,19 +146,6 @@ class ActivityServiceTest {
       )
     }
 
-    @Test
-    fun `telemetry event is created`() {
```
Comment (mikehalmamoj) [inline src/test/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/activities/ActivityServiceTest.kt:151]: Moved to integration test to check the ids

File: src/test/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/sentencing/SentencingAdjustmentsResourceIntTest.kt [hunk 2]
```diff
@@ -36,9 +35,6 @@ class SentencingAdjustmentsResourceIntTest : IntegrationTestBase() {
   @SpyBean
   private lateinit var spRepository: StoredProcedureRepository
 
-  @SpyBean
-  private lateinit var telemetryClient: TelemetryClient
```
Comment (mikehalmamoj) [inline src/test/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/sentencing/SentencingAdjustmentsResourceIntTest.kt:40]: Pushed up to base class

File: src/test/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/activities/ActivityResourceIntTest.kt [hunk 3]
```diff
@@ -246,6 +253,29 @@ class ActivityResourceIntTest : IntegrationTestBase() {
...
+        eq("activity-created"),
+        check<MutableMap<String, String>> { actual ->
+          mapOf(
+            "courseActivityId" to id.toString(),
+            "prisonId" to PRISON_ID,
+            "courseScheduleIds" to "[${courseActivity.courseSchedules[0].courseScheduleId}]",
+            "courseActivityPayRateIds" to "[BAS-5-2022-10-31]",
+            "courseScheduleRuleIds" to "[${courseActivity.courseScheduleRules[0].id}]",
+          ).also { expected ->
+            log.info("expected telemetry details to be: $expected")
```
Comment (mikehalmamoj) [inline src/test/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/activities/ActivityResourceIntTest.kt:271]: This test was failing but not printing the expected values! So I'm logging them manually 🤷

**Review by petergphillips — APPROVED** [review]
LGTM

**Pull request merged**